pytorch/inference/docker/1.5.1/py3/Dockerfile.neuron

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:16.04
+FROM ubuntu:18.04
 
 LABEL dlc_major_version="1"
```

Comment:
If we do need to support an ubuntu 18.04 variant of the PT 1.5.1 Neuron DLC, could you please create a separate dockerfile for it, possibly named `Dockerfile.ubuntu18.04.neuron`, which contains these changes?

In the new dockerfile, please change line 110 to constrain torch-neuron to `"torch-neuron>=1.5.1.1.3,<1.5.2"` instead of `torch-neuron>=1.5.1,<1.5.2`. Please also change `dlc_major_version` from `"1"` to `"2"` in the new dockerfile, in line 3.

In the existing ubuntu16.04 dockerfile, please change line 110 to constrain torch-neuron to `"torch-neuron>=1.5.1.1.2,<1.5.1.1.3"` instead of `torch-neuron>=1.5.1,<1.5.2`.